### DISCUSSION TITLE
图片脚注和图片不在同一页

### DISCUSSION THREAD

**Discussion by Chenyanlu99** [TheNetAdmin/zjuthesis#437] [General]
代码如下：
`\begin{figure}[h]`
`	\centering`
`	\includegraphics[width=0.8\textwidth]{xxx.png}`
`	\caption{xxxxxx\protect\footnotemark}`
`	\label{Fig:xxx}%`
`\end{figure}`
`\footnotetext{note1}`
想在图片标题上添加脚注序号，脚注内容在改页左下角显示。但是由于浮动体的排布原因，可能出现脚注在第3页，图片在第4页的情况，脚注比脚注编号先出现肯定是不合理的，请问这种情况有办法解决吗？

**Comment by xfz329** [comment]
适当调整图片之前的内容吧。没有复现问题的代码，没法细debug
